lib/one_model.ex

```diff
@@ -163,7 +163,7 @@ defmodule OneModel do
         * Passing a list of keys will return the default schema with the values for the specified keys.
       * `limit: integer`
       """
-      @spec list(keyword()) :: [t()]
+      @spec list(keyword()) :: any
```

Comment:
It's still going to return a list, so let's no drop that part of the contract.
```suggestion
      @spec list(keyword()) :: [any]
```

Reply:
Or as Steve and I discussed yesterday
```suggestion
      @spec list(keyword()) :: list
```

Reply:
I used `iist`